components/style/Cargo.toml

```diff
@@ -79,7 +80,7 @@ void = "1.0.2"
 
 [build-dependencies]
 lazy_static = "1"
-log = "0.4"
+#log = "0.4"
```

Comment:
Was this supposed to be removed?

Reply:
I didn’t mean to commit this. Reverted, thanks.

`log` is used unconditionally in the library crate, but only in Gecko mode in the build script. So having this in `build-dependencies` warns in Servo mode. I didn’t bother to try if Cargo supports having a dependency optional in `build-dependencies` but non-optional in `depnedencies`.